src/core/service/notifications/registry/handler.go

```diff
@@ -173,12 +173,11 @@ func filterEvents(notification *models.Notification) ([]*models.Event, error) {
 }
 
 func checkEvent(event *models.Event) bool {
-	// pull and push manifest by docker-client or vic or jib
-	if (strings.HasPrefix(event.Request.UserAgent, "docker") || strings.HasPrefix(event.Request.UserAgent, vicPrefix) || strings.HasPrefix(event.Request.UserAgent, "jib")) &&
-		(event.Action == "pull" || event.Action == "push") {
+	// pull and push manifest
+	if strings.ToLower(strings.TrimSpace(event.Request.UserAgent)) != "harbor-registry-client" && (event.Action == "pull" || event.Action == "push") {
```

Comment:
is this code change caused by a wrong rebase action?

Reply:
This is not a change I made.